lib/find.coffee

```diff
@@ -115,6 +115,16 @@ module.exports =
       'find-and-replace:select-skip': (event) ->
         selectNextObjectForEditorElement(this).skipCurrentSelection()
 
+    # Request autocompletions for the find field.
+    autocompletePackage = atom.packages.activePackages['autocomplete-plus']
+    if autocompletePackage?
+      autocompleteManager = autocompletePackage.mainModule.autocompleteManager
+      for providerData in autocompleteManager.providerManager.providers
+        if providerData.provider.constructor.name is 'SymbolProvider'
```

Comment:
This is a bit ugly, but I think it's okay for now. In the future, providers could ideally be registered together with a name.
It would also allow you to register multiple instances of the same provider with different options e.g. an `ActiveBufferSymbolProvider`, `AllBuffersSymbolProvider`, `ProjectSymbolProvider`, etc.